src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,169 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capacity;
+    private int threshold;
+    private int size;
+    private double loadFactor;
+    private Node<K, V>[] table;
+    private final int defaultInitialCapacity = 16;
+    private final double defaultLoadFactor = 0.75f;
```

Comment:
these should be constants , Don't forget to use CONSTANT_CASE

Reply:
OK